deployment/helm/values.yaml

```diff
@@ -109,7 +109,7 @@ debugEcho:
     # container used by debugEcho
     image: nginx:latest
     # pullPolicy is the debugEcho pull policy
-    pullPolicy: Always
+    pullPolicy: ""
```

Comment:
not sure i know what happens if this empty value is passed into a PodSpec.

we may want to update each occurence of `imagePullPolicy: {{ .Values.*.image.pullPolicy }}` to be something like 

{{- with .Values.*.image.pullPolicy }}
imagePullPolicy: {{ .Values.*.image.pullPolicy }}
{{- end }}

so that the imagePullPolicy is only configured if someone sets a value.

Reply:
I tried it out with the following dummy deployment and it defaulted to IfNotPresent behavior
```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: pull-policy-deployment
  labels:
    app: pull-policy
spec:
  selector:
    matchLabels:
      app: pull-policy
  template:
    metadata:
      labels:
        app: pull-policy
    spec:
      containers:
      - name: pull-policy
        image: ghcr.io/kate-goldenring/pull-policy:latest
        imagePullPolicy: ""
```

Reply:
But that seems to contradict this line in their documentation: "When imagePullPolicy is defined without a specific value , it is also set to Always", so maybe I should either test again or go with what you're suggesting

Reply:
It says "Container image "ghcr.io/kate-goldenring/pull-policy:v0" already present on machine" when i deploy for the second time. Then, if I change to the latest tag, it pulls it: "Pulling image "ghcr.io/kate-goldenring/pull-policy:latest""

Reply:
looks like you're getting the behavior you want ... i don't feel super strongly, but if it were my PR, i'd probably avoid `imagePullPolicy: ""`.

Reply:
Looks much cleaner your way! Fixed